documentation/modules/security/proc-renewing-ca-certs-manually.adoc

```diff
@@ -27,16 +30,21 @@ A renewed certificate uses the same private key as the old certificate.
 ¦Annotate command
 
 ¦Cluster CA
-¦_<cluster-name>_-cluster-ca-cert
-m¦kubectl annotate secret _<cluster-name>_-cluster-ca-cert strimzi.io/force-renew=true
+¦_KAFKA-CLUSTER-NAME_-cluster-ca-cert
+m¦kubectl annotate secret _KAFKA-CLUSTER-NAME_-cluster-ca-cert strimzi.io/force-renew=true
 
 ¦Clients CA
-¦_<cluster-name>_-clients-ca-cert
-m¦kubectl annotate secret _<cluster-name>_-clients-ca-cert strimzi.io/force-renew=true
+¦_KAFKA-CLUSTER-NAME_-clients-ca-cert
+m¦kubectl annotate secret _KAFKA-CLUSTER-NAME_-clients-ca-cert strimzi.io/force-renew=true
 
 |===
 
-At the next reconciliation the Cluster Operator will generate a new CA certificate for the `Secret` that you annotated. 
+. Delete old certificates from the Secret.
++
+When components are using the new certificates, older certificates might still be active.
+Delete the old certificates to remove any potential security risk.
```

Comment:
I think this should only at the end of the file: before additional resources. Or it should be embedded as a `NOTE:` or something. Because this should be done only after the renewal etc.

Reply:
I think a proper step is less likely to be missed than a note. How about this:

![image](https://user-images.githubusercontent.com/47596553/90126683-f75c3880-dd5b-11ea-90c5-2216345d7a09.png)